donkeycar/parts/datastore.py

```diff
@@ -236,6 +236,13 @@ def put_record(self, data):
                 img.save(os.path.join(self.path, name))
                 json_data[key]=name
 
+            elif typ == 'depth_array':
```

Comment:
Nit: call this something like `np_array` so we can support arbitrary array types going forward.

Reply:
It's not totally generic.  This line `img = Image.fromarray(np.uint16(val))` requires a 16 bit array as input.  Then it always outputs that to a png.  so we could change it be more generic a name, but we want to keep the semantics of the constraint.  So something like 'gray16_array' or 'np16_array'.